frontend/src/bundles/talent-onboarding/components/skills-step/components/close-icon/close-icon-button.tsx

```diff
@@ -0,0 +1,25 @@
+import { Close } from '@mui/icons-material';
+
+import { useCallback } from '~/bundles/common/hooks/hooks.js';
+
+import styles from './styles.module.scss';
+
+type Properties = {
+    index: number;
+    onClick: (index: number) => void;
+};
+const CloseIconButton: React.FC<Properties> = ({ index, onClick }) => {
+    const handleClick = useCallback(() => {
+        onClick(index);
+    }, [index, onClick]);
+
+    return (
+        <Close
+            color="error"
```

Comment:
This color is not the same we already use for errors text in our forms:
Color of icon: #d32f2f;
Color of error text:  #ff2121

Reply:
fixed